### DISCUSSION TITLE
Where to submit bugs for issues that are not brew specific issues???

### DISCUSSION THREAD

**Discussion by oregonpillow** [Homebrew/discussions#4743] [Everyday usage]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

I found a bug on website  that fails to render a picture in documentation despite the src link being valid. How am i supposed to report this? 

The only bug issue template (https://github.com/Homebrew/brew/issues/newassignees=&labels=bug&projects=&template=bug.yml) makes it very clear you'll be crucified for not filling in the issue template completely despite all the forms not being relevant to this.

**Comment by SMillerDev** [comment]
What website? [brew.sh](https://github.com/Homebrew/brew.sh)? [formulae.brew.sh](https://github.com/Homebrew/formulae.brew.sh)? Something else?

**Reply by oregonpillow** [reply]
https://docs.brew.sh/Brew-Test-Bot

**Reply by SMillerDev** [reply]
That's probably still brew.sh

**Reply by oregonpillow** [reply]
Thanks for some reason i didn't find the brew.sh repo, i found it now. Thanks! 
https://github.com/Homebrew/brew.sh
